### PR TITLE
Theme Designer: starting to build out styles for layout

### PR DESCRIPTION
**Pull request by emmayjiang**
Adding styles to the preexisting layout of the theme designer such that it now has a designated area for the navbar, sidebar, and content.

image

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/3321df44cdc17b93c110019f7365bc80f0004e1b'>3321df44cdc17b93c110019f7365bc80f0004e1b</a></sub>
<div id="bundleSizeComment9423"/>

File: packages/react-components/theme-designer/src/components/Nav/Nav.tsx [hunk 1]
```diff
@@ -1,9 +1,22 @@
...
 }
 
+const useStyles = makeStyles({
+  root: {
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'left',
+    borderBottomWidth: '1px',
+    borderBottomColor: '#D1D1D1',
+    borderBottomStyle: 'solid',
```
Comment (micahgodbolt) [inline packages/react-components/theme-designer/src/components/Nav/Nav.tsx:15]: I forgot to mention this on the parent container. but you can use shorthands to simplify this
```suggestion
   ...shorthands.borderBottom('1px', 'solid', '#D1D1D1')
```

File: packages/react-components/theme-designer/src/components/Component/Component.stories.tsx [hunk 2]
```diff
@@ -1,3 +0,0 @@
-import { Component } from './Component';
```
Comment (micahgodbolt) [inline packages/react-components/theme-designer/src/components/Component/Component.stories.tsx:1]: you don't have to keep these around if you don't want to. your call.

**Review by micahgodbolt — CHANGES_REQUESTED** [review]
few small tweaks with interface and styles

**Review by micahgodbolt — APPROVED** [review]

**Pull request merged**